sdk/python/kfp/components/structures.py

```diff
@@ -154,8 +270,8 @@ class InputUriPlaceholder(base_model.BaseModel, PlaceholderSerializationMixin):
     _FROM_PLACEHOLDER_REGEX = r"\{\{\$\.inputs\.artifacts\[(?:''|'|\")(.+?)(?:''|'|\")]\.uri\}\}"
 
 
-class OutputParameterPlaceholder(base_model.BaseModel,
-                                 PlaceholderSerializationMixin):
+class OutputValuePlaceholder(base_model.BaseModel,
```

Comment:
`OutputParameterPlaceholder` is probably a better name. We don't have a `{outputValue: }` placeholder per se. The placeholder is still an **output (file) path** (as indicated by `.output_file`) for parameter.

Reply:
Thanks, @chensun. Updated!